shared/bash_remediation_functions/disable_prelink.sh

```diff
@@ -3,10 +3,10 @@ function disable_prelink {
 	# whether it is installed.
 	if grep -q ^PRELINKING /etc/sysconfig/prelink
 	then
-		sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
+		sed -i 's/^PRELINKING[:blank:]*=[:blank:]*[:alpha:]*/PRELINKING=no/' /etc/sysconfig/prelink
 	else
-		printf '\n%s\n' '# Set PRELINKING=no per security requirements' >> /etc/sysconfig/prelink
-		echo 'PRELINKING=no' >> /etc/sysconfig/prelink
+		printf '\n'
```

Comment:
did you mean printf into the file? this is not redirected to it

Reply:
Sure, I have fixed the missing redirect in rebase.

Reply:
Have you pushed the rebase? It still shows up in the hunk.

Reply:
Yes, finally.